Sources/MongoMobile/MongoMobile.swift

```diff
@@ -140,6 +134,17 @@ public class MongoMobile {
         if let cachedInstance = embeddedInstances[settings.dbPath] {
             instance = cachedInstance
         } else {
+            // NOTE: This hack can be removed once SERVER-38943 is resolved. Also note
+            //       that the destroy code is not refactored into a common function because
+            //       we should be removing this code in the near future.
+            if embeddedInstances.count > 0 {
```

Comment:
I realize this will be removed soon so I don't care about correcting this particular linter violation (`empty_count` rule), but it does make me think of the more general point that maybe we should copy over our linter rules? (could be done in a follow up PR.)

Reply:
yep, I was going to suggest the same. I'll add it in here now